workflows/utils/tantalus_utils.py

```diff
@@ -63,3 +67,78 @@ def get_upstream_datasets(results_ids):
     return list(upstream_datasets)
 
 
+def create_analysis(analysis_type, jira_ticket, version, args):
+    """
+    Creates analysis object on tantalus with minimal fields
+
+    Args:
+        analysis_type (str): type of analysis
+        jira_ticket (str): jira ticket key ex. SC-####
+        version (str): version of singlecellpipeline ex. v#.#.#
+        args (dict): analysis info 
+    """
+    # get align analysis in order to get template for analysis name
+    align_analysis = tantalus_api.get(
+        "analysis",
+        jira_ticket=jira_ticket,
+        analysis_type__name="align",
+    )
+
+    # replace align in name with analysis type
+    name = align_analysis["name"].replace("align", analysis_type)
+
+    # set fields and keys
+    fields = dict(
+        name=name,
+        analysis_type=analysis_type,
+        version=version,
+        jira_ticket=jira_ticket,
+        args=args,
+        status="ready",
+    )
+
+    keys = ['name']
+
+    # create analysis
+    analysis, _ = tantalus_api.create('analysis', fields, keys, get_existing=True)
+
+
+def create_qc_analyses_from_library(library_id, jira_ticket, version, analysis_type):
+    """ 
+    Create align, hmmcopy, and annotation analysis objects
+
+    Args:
+        library_id (str): library name
+        jira_ticket (str): jira ticket key ex. SC-####
+        version (str): version of singlecellpipeline ex. v#.#.#
+    """
+
+    # get library info from colossus
+    library = colossus_api.get('library', pool_id=library_id)
+    reference_genome = get_ref_genome(library)
+
+    # add arguments
+    args = {}
+    args['library_id'] = library_id
+    # default aligner is BWA_MEM_0_7_6A
+    args['aligner'] = "BWA_MEM_0_7_6A"
```

Comment:
The aligner version is problematic here, the aligner has been updated in the pipeline but we are still creating records using 0_7_6A as i undertstand.  Ideally we just specify mem and then the actual version gets pulled out of the results.

Reply:
For future rework im guessing